uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/service/UniProtEntryService.java

```diff
@@ -314,4 +319,14 @@ private boolean needToAddActiveFilter(UniProtKBSearchRequest uniProtRequest) {
     private String getQueryFieldName(String active) {
         return searchFieldConfig.getSearchFieldItemByName(active).getFieldName();
     }
+
+    private ProblemPair getLeadingWildcardIgnoredWarning(String query) {
+        if (SolrQueryUtil.ignoreLeadingWildcard(
+                query, this.uniProtQueryProcessorConfig.getLeadingWildcardFields())) {
+            return new ProblemPair(
+                    41,
+                    "Leading wildcard (*, ?) was removed for this search. Please check the help page for more information on using wildcards on queries.");
```

Comment:
I think we have an enum to group all these messages, right? Do you think we could use it here? An even better solution would be to use message.properties for it.